### DISCUSSION TITLE
Created a second account with email login - now I cannot access my first account with login via Github

### DISCUSSION THREAD

**Discussion by oliverjohns** [vercel/community#5616] [Account Issue] [answered]
### Question

Years back I created an account on vercel, Account A, which is created via github sign in

Today I created a second account, Account B, by adding an email. I then proceeded to importing repositories from the same github account that account A uses.

Now I cannot login to account A with github anymore, it only takes me to account B!

I need to be able to login to Account A with "Sign in with Github" like I have always done. Right now I'm locked out of my account.

This is very urgent. I appreciate all the help I can get. Thanks!

**Comment by amyegan** [comment, marked as answer]
Hi @oliverjohns. You can only sign in to one account via GitHub, so the connection with Account A was severed when you connected to Account B. 

You can still access Account A by logging in with email instead of GitHub. Please let me know if you run into trouble with that 🙏
